### PR TITLE
Clean better.

### PR DESCRIPTION
**Pull request by pphaneuf**

### CODE REVIEW

**Comment by phad** [comment]
Did it in fact previously do no cleaning at all?

File: Makefile.am [hunk 1]
```diff
@@ -19,6 +19,10 @@ BUILT_SOURCES = \
 	proto/ct.pb.cc \
 	proto/ct.pb.h
 
+CLEANFILES = \
```
Comment (phad) [inline Makefile.am]: Is this a standard Makefile target?

Comment (pphaneuf) [inline Makefile.am]: This is an Automake variable: https://www.gnu.org/software/automake/manual/automake.html#Clean

Comment (phad) [inline Makefile.am]: ok thx

**Comment by pphaneuf** [comment]
There was cleaning, but only of what Automake knew about (`.o` files, binaries, that sort of things). Protocol buffer generated output (`protoc` stuff) isn't known by Automake, so it wasn't cleaned, and the version.cc file that I generated with a rule nearby.

Not a big deal, we rarely do `make clean`, but if we do it, we'd expect all generated files to get deleted.

**Pull request merged**